akka-docs/src/main/paradox/typed/coexisting.md

```diff
@@ -116,13 +116,20 @@ Scala
 Java
 :  @@snip [TypedWatchingClassicTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/TypedWatchingClassicTest.java) { #typed }
 
+Note that when sending from a typed actor to a classic @scala[`ActorRef`]@java[`ActorRef`] there is no sender in scope as in classic.
```

Comment:
I think this paragraph also belongs inside the scala group. For java it's alway explicit `tell(msg, sender)`

Reply:
Oops, thanks.